cmd/rook/ceph/cleanup.go

```diff
@@ -40,22 +42,41 @@ var (
 
 var cleanUpCmd = &cobra.Command{
 	Use:   "clean",
-	Short: "Starts the cleanup process on the disks after ceph cluster is deleted",
+	Short: "Starts the cleanup process",
+}
+
+var cleanUpHostsCmd = &cobra.Command{
+	Use:   "hosts",
```

Comment:
Only one host is cleaned up by this command for a particular cleanup job, right?
```suggestion
	Use:   "host",
```